docs/DeveloperGuide.md

```diff
@@ -567,64 +567,211 @@ testers are expected to do more *exploratory* testing.
+
+`listPerson Tan`
+
+Expected Output in the Person List Panel: Nothing is displayed.
+
+Expected Output in the Command Output Panel: List person message is shown and states "0 person(s) listed".
+
+### Settling an outstanding balance with a Person
+
+Prerequisite: There is only 1 person stored in Spend N Split. Their name is "Alex" and their balance shown is +20.00.
+
+`settlePerson 1`
+
+Expected Output in the Transaction List Panel: A new transaction is created that is named "Settle balance with Alex".
+
+Expected Outp t in the Person List Panel: The balance of Alex is now "0.00"
+
+Expected Output in the Command Output Panel: Balance settled message is shown along with the name of Alex.
+
+### Sorting people by their balances
+
+Prerequisite: There are a few people stored in Spend N Split with varying balances.
+
+`sortPerson +`
+
+Expected Output in the Person List Panel: The people displayed are sorted in descending order of balance.
+
+Expected Output in the Command Output Panel: A message stating that the Person list is sorted in descending order of 
+balance is shown.
+
+`sortPerson -`
+
+Expected Output in the Person List Panel: The people displayed are sorted in ascending order of balance.
+
+Expected Output in the Command Output Panel: A message stating that the Person list is sorted in ascending order of
+balance is shown.
+
+### Adding a transaction
+
+Prerequisite: There is only 1 person stored in Spend N Split. Their name is "Alex Yeoh".
+
+`addTransaction d=bread n=Alex Yeoh c=25.00 n=Self w=1 n=Alex Yeoh w=1`
+
+Expected Output in the Transaction List Panel: New transaction is added into the list.
+
+Expected Output in the Command Output Panel: New transaction added message is shown along with the transaction's details.
+
+
+### Deleting a Transaction
+
+Prerequisite: There is at least 1 transaction in the Transaction List Panel.
+
+`deleteTransaction 1`
+
+Expected Output in the Transaction List Panel: The first transaction in the list is deleted.
+
+Expected Output in the Command Output Panel: Delete transaction message is shown along with the transaction's details.
+
+`deleteTransaction 0`
+
+Expected Output in the Command Output Panel: Error message stating that there is an invalid command format.
+### Editing a Transaction's details
+
+Prerequisite: There is at least 1 transaction in the Transaction List Panel.
+
+`editTransaction 1 c=20`
+
+Expected Output in the Transaction List Panel: The first transaction in the list has their cost changed to 20.00, and retains
+the rest of their details.
+
+Expected Output in the Command Output Panel: Edit transaction message is shown along with the transaction's details.
+
+`editTransaction 0`
+
+Expected Output in the Command Output Panel: Error message stating that there is an invalid command format.
+
+### Listing multiple transactions with matching payers and payees.
+
+Prerequisite: There is only 1 transaction stored in Spend N Split. The payer is "Roy".
+
+`listTransaction Roy`
```

Comment:
The format should be n=Roy, if n= is not specified, the transaction description is searched instead

Reply:
fixed